test/inductor/test_torchinductor.py

```diff
@@ -604,6 +604,54 @@ def fn(x, y, z):
 
         self.common(fn, (x, y, z))
 
+    # lowering operation tests
+    def test_add_uint8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([128], dtype=torch.uint8)
+
+        self.common(fn, x)
+
+    def test_add_int8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([120], dtype=torch.int8)
+
+        self.common(fn, x)
+
+    def test_add_int16_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int32)
+
+        x = torch.tensor([35000], dtype=torch.int16)
+
+        self.common(fn, x)
+
+    def test_multiply_uint8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([8], dtype=torch.uint8)
+
+        self.common(fn, x)
+
+    def test_square_uint8_tensor(self):
+        def fn(x):
+            return (x**2).to(torch.int16)
+
+        x = torch.tensor([8], dtype=torch.uint8)
```

Comment:
```suggestion
        x = torch.tensor([32], dtype=torch.uint8)
```

Reply:
The square root of 256 is 16, not 8. Also (32 * 32 % 256) is also 0.

Reply:
ops, I wanted to put any large number, but "32" is not a great one. My point was that 8**2 does not overflow here, so it's always going to be correct unless we do something very wrong

Reply:
@peterbell10 @lezcano oops.
 I will correct!